module-db/tests/ContactsRecord_tests.cpp

```diff
@@ -760,3 +760,163 @@ TEST_CASE("Check if new contact record exists in DB as a temporary contact")
         REQUIRE(records.verifyTemporary(noTemporaryContactRecord) == true);
     }
 }
+
+TEST_CASE("Check replacement of number in place in db when only different is having of country code")
+{
+    db::tests::DatabaseUnderTest<ContactsDB> contactsDb{"contacts.db", db::tests::getPurePhoneScriptsPath()};
+
+    // Preparation of DB initial state
+    auto records = ContactRecordInterface(&contactsDb.get());
+    ContactRecord testContactRecord;
+
+    std::array<std::string, 2> numbers   = {{{"500600100"}, {"500600200"}}};
+    std::array<std::string, 2> numbersPL = {{{"+48500600100"}, {"+48500600200"}}}; // Poland country code
+    std::array<std::string, 2> numbersFR = {{{"+33500600100"}, {"+33500600200"}}}; // France country code
+
+    SECTION("Temporary contact exists but new one with same number have country code")
+    {
+        // Number of record in number table should stay the same
+        constexpr int32_t NUMBER_IN_RECORD_DURING_REPLACEMENT = 1;
+
+        /*Adding of temporary contact without country code*/
```

Comment:
Why are some one-line comments in another style?

Reply:
`/**/` style is use to comment more code placed below this comment. It's like comment for whole code section below.
`// `style touch only specific line

What you think about that? 
Maybe it's not as clear as I thought. hmmm.

Reply:
My understanding was that `/**/` was used for multiline comments and `//` for one-liners. `/**/` not defines which part of the code it is documenting (to the first blank line? next definition? next indentation?).

Reply:
@rrandomsky I have to agree with @jurgenwigg - I have the same conclusions. AFAIK we don't have specified those rules in our docs

Reply:
here you can find a guideline - mostly based on ISO CPP:
https://github.com/mudita/MuditaOS/blob/master/doc/MuditaCppCodingGuidelines.md

Regarding python - I'd say PEP8 is the best guideline for our purposes :)
I'll answer your QA PR's today - don't feel abandoned :P

Reply:
Ok :) no problem. I've also reported some issues (I followed the contribution guide to report them first instead of just creating PR for that) which I can implement ;)

Reply:
Ok, thank you for your opinions.
I changed all `/**/` to `//`. 
Please, let me note if there is something else :)